docs/release-notes/RELEASE-NOTES-0.16.0.md

```diff
@@ -2,6 +2,8 @@
 
 [This release][this-release] is focused on TODOFILLMEIN.
 It:
+* Improves TX-pool synchronization. It is now synced (with only missing transactions being
```

Comment:
What if the node gets out of sync for some reason (e.g. temporary connection issue)? Does it need to be restarted to catch up?

Reply:
First, it will still work, although potentially missing some transactions is an issue if it becomes the leader (otherwise it will pull missing transactions when a block contains them). 

The tricky thing is for the node to realize it is out of synch, if it can do this it should be straightforward to re-synch (adding an API for this is 10 minutes work)... Might be worth thinking about.